### ISSUE TITLE
Won't start after updating to .9

### ISSUE THREAD

**Issue by vikingkarwur** [brave/browser-laptop#1298]
![screen shot 2016-04-09 at 14 53 02](https://cloud.githubusercontent.com/assets/464974/14402544/4a4e6e96-fe63-11e5-8408-5a5c30cc518c.png)
![screen shot 2016-04-09 at 14 52 52](https://cloud.githubusercontent.com/assets/464974/14402545/4a99ac76-fe63-11e5-83cf-9afdfce28b2c.png)

**Comment by loranger** [comment]
Same here.

Here are the error message in text format. The first one :

--- 
__A JavaScript error occurred in the main process__
```
Uncaught Exception:
TypeError: Parameter 'url' must be a string, not object
    at Url.parse (url.js:73:11)
    at urlParse (url.js:67:5)
    at data.passwords.forEach (/opt/homebrew-cask/Caskroom/brave/0.8.3dev/Brave.app/Contents/Resources/app.asar/app/sessionStore.js:250:19)
    at Array.forEach (native)
    at fs.readFile.e (/opt/homebrew-cask/Caskroom/brave/0.8.3dev/Brave.app/Contents/Resources/app.asar/app/sessionStore.js:249:24)
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:404:3)
```

--- 
And the second one :

---
__A JavaScript error occurred in the main process__
```
Uncaught Exception:
TypeError: Cannot read property 'get' of undefined
    at setDefaultWindowSize (/opt/homebrew-cask/Caskroom/brave/0.8.3dev/Brave.app/Contents/Resources/app.asar/js/stores/appStore.js:222:16)
    at windowDefaults (/opt/homebrew-cask/Caskroom/brave/0.8.3dev/Brave.app/Contents/Resources/app.asar/js/stores/appStore.js:197:3)
    at process.handleAppAction (/opt/homebrew-cask/Caskroom/brave/0.8.3dev/Brave.app/Contents/Resources/app.asar/js/stores/appStore.js:250:52)
    at emitOne (events.js:90:13)
    at process.emit (events.js:182:7)
    at AppDispatcher.dispatch (/opt/homebrew-cask/Caskroom/brave/0.8.3dev/Brave.app/Contents/Resources/app.asar/js/dispatcher/appDispatcher.js:18:15)
    at Object.appActions.newWindow (/opt/homebrew-cask/Caskroom/brave/0.8.3dev/Brave.app/Contents/Resources/app.asar/js/actions/appActions.js:32:19)
    at EventEmitter.app.on (/opt/homebrew-cask/Caskroom/brave/0.8.3dev/Brave.app/Contents/Resources/app.asar/app/index.js:229:18)
    at emitTwo (events.js:105:20)
    at EventEmitter.emit (events.js:185:7)
```
---

(I know it looks like an 0.8.3dev, but it was just updated in 0.9)

**Comment by loranger** [comment]
Deleting `~/Library/Application Support/brave/session-store-1` fixed the issue, but all is lost (passwords, bookmarks and previous session).

At least, I can browse the web again.

**Comment by bbondy** [comment]
@bridiver I seen this in support as well from some people. 
<img width="583" alt="screenshot 2016-04-09 11 28 06" src="https://cloud.githubusercontent.com/assets/831718/14404513/40db8b40-fe46-11e5-86b7-367d00542747.png">

@vikingkarwur it happens on each startup now?

**Comment by bbondy** [comment]
Will be fixed in 0.9.2 instead as we have steps now.
Steps here:
https://github.com/brave/browser-laptop/commit/59afbe86e08705259ba3061efb9b347700805cc6#commitcomment-17042506

Also that can be reverted once we get the real fix.

The workaround is just to quit Brave and then open it again.  It happens when clicking multiple times in a row when opening the app.  You have to quit the app from the menu and open it again so the error goes away.
